cmd/notary/tuf.go

```diff
@@ -368,19 +365,13 @@ func getTransport(gun string, readOnly bool) http.RoundTripper {
 		if !filepath.IsAbs(rootCAFile) {
 			rootCAFile = filepath.Join(configPath, rootCAFile)
 		}
-		rootCert, err := trustmanager.LoadCertFromFile(rootCAFile)
-		if err != nil {
-			fatalf("could not load root ca file. %s", err.Error())
-		}
-		rootPool.AddCert(rootCert)
 	}
 
-	// skipTLSVerify is false by default so verification will
-	// be performed.
-	tlsConfig := &tls.Config{
-		InsecureSkipVerify: mainViper.GetBool("remote_server.skipTLSVerify"),
-		MinVersion:         tls.VersionTLS10,
-		RootCAs:            rootPool,
+	tlsConfig, err := utils.ConfigureClientTLS(
+		rootCAFile, "", mainViper.GetBool("remote_server.skipTLSVerify"),
```

Comment:
What causes `remote_server.skipTLSVerify` to default to `false`? I think I'd rather see this value set to false here and then only set if the viper parser passes us something. Seems better than relying on viper to default values to false. (I don't see a default explicitly being set for this, sorry if I missed it)